packages/core/src/internal/channel.js

```diff
@@ -156,10 +156,7 @@ export function multicastChannel() {
     closed = true
     const takers = (currentTakers = nextTakers)
 
-    for (let i = 0; i < takers.length; i++) {
-      const taker = takers[i]
-      taker(END)
-    }
+    takers.forEach(taker => taker(END))
```

Comment:
we dont care about return value, so it's better to wrap the body in curly braces
```js
takers.forEach(taker => { taker(END) })
```